internal/cephfs/volumemounter.go

```diff
@@ -41,7 +41,7 @@ const (
 var (
 	availableMounters []string
 
-	// maps a mountpoint to PID of its FUSE daemon
+	// maps a mountpoint to PID of its FUSE daemon.
```

Comment:
if this is needed, put it in its own commit please

Reply:
its needed for linter to be happy. Will move to seperate commit while organizing the commits.

Reply:
if this is needed how the current CI is passing?